### PR TITLE
Add Spin Cloud SQLite Reference (v0.1.2)

### PR DESCRIPTION
**Pull request by mjia8**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [X] The `title, `template`, and `date` are all set
- [X] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [X] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-07-26 at 12 21 03 pm

Screenshot 2023-07-26 at 12 20 57 pm

- [X] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-07-26 at 12 22 00 pm

Screenshot 2023-07-26 at 12 22 28 pm

- [X] Headings are using Title Case
- [X] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [X] Have tested with `npm run test` and resolved all errors

Screenshot 2023-07-26 at 12 21 49 pm

- [X] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

**Embargo**

embargo

### CODE REVIEW

File: content/cloud/cloud-command-reference.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-comman
 - [spin cloud deploy](#spin-cloud-deploy)
 - [spin cloud help](#spin-cloud-help)
 - [spin cloud login](#spin-cloud-login)
+- [spin cloud sqlite](#spin-cloud-sqlite)
+- [spin cloud sqlite delete](#spin-cloud-sqlite-delete)
+- [spin cloud sqlite help](#spin-cloud-sqlite-help)
+- [spin cloud sqlite list](#spin-cloud-sqlite-list)
 - [spin cloud variables](#spin-cloud-variables)
```
Comment (macolso) [inline content/cloud/cloud-command-reference.md:18]: we will have one more command (@rylev  is currently working on it and can give us a thumbs up when its ready for us to copy/paste into dev docs)
- [spin cloud sqlite help](#spin-cloud-sqlite-execute)

Comment (tpmccallum) [reply content/cloud/cloud-command-reference.md:18]: Thanks @macolso 
I think I can see the new PR that @rylev is about to have approved and merged. It is at https://github.com/fermyon/cloud-plugin/pull/49

My assumption is that once this new PR is merged we can again use the latest version of Canary via this installation command `spin plugin install --url https://github.com/fermyon/cloud-plugin/releases/download/canary/cloud.json` and then do the `spin cloud sqlite --help` to see what new subcommands and their subcommands may be.

The new `execute` command (I think that is the subcommand) can be added to this PR.

Comment (tpmccallum) [reply content/cloud/cloud-command-reference.md:18]: @mjia8 could try and access this new subcommand on the new release of canary and update this PR.

**Comment by tpmccallum** [comment]
This is complete now and ready for approval.

**Comment by tpmccallum** [comment]
There is no Spin `1.5` release; because that will come after Berlin.
This PR is in anticipation of the new cloud plugin release which has the sqlite feature.

**Review by macolso — APPROVED** [review]
LGTM

**Review by tpmccallum — APPROVED** [review]
Thank you @mjia8 
This is great, I am going to take the `1.5` content that you created and put it aside for when we inevitably go from `1.4` to `1.5`.

**Pull request merged**